components/navigation/DocsMobileMenu.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import type { DocsMobileMenuProps } from '@/types/navigation/DocsMobileMenu';
```

Comment:
Define these Props types in the component file itself, if we don't have to use them anywhere else. Do it for every file.